imageio/plugins/ffmpeg.py

```diff
@@ -471,24 +510,41 @@ def _open(self, fps=10, codec='libx264', bitrate=400000):
             # Initialize parameters
             self._proc = None
             self._size = None
-        
-        def _close(self):
-            # Close subprocess
-            if self._proc is not None:
-                self._proc.stdin.close()
-                self._proc.stderr.close()
-                self._proc.wait()
-                self._proc = None
-        
+            self._cmd = None
+
+        def _close(self, timeout=1.0):
+            # Somtimes ffmpeg process not getting closed.
+            # This attempts to reliably close the process.
+            if self._proc is None:  # pragma: no cover
+                return  # no process
+            if self._proc.poll() is not None:
+                return  # process already dead
+            # Terminate process
+            self._proc.stdin.close()
+            self._proc.wait()
+            if self._proc.poll() is not None:
+                return  # process already dead
+            self._proc.terminate()
```

Comment:
I think `proc.wait()` would wait for it to stop, so anything that comes below is not necessary... or maybe we should not call `proc.wait()` at all?